logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/conversation/message/MessageUnpackResult.kt

```diff
@@ -0,0 +1,20 @@
+package com.wire.kalium.logic.sync.receiver.conversation.message
+
+import com.wire.kalium.logic.data.conversation.ClientId
+import com.wire.kalium.logic.data.id.ConversationId
+import com.wire.kalium.logic.data.message.ProtoContent
+import com.wire.kalium.logic.data.user.UserId
+
+internal sealed interface MessageUnpackResult {
+
+    object ProtocolSignalingMessage : MessageUnpackResult
```

Comment:
I would maybe call this `HandshakeMessage`, but we don't use this type for anything so why not remove it and return an optional  `ApplicationMessage` instead.

Reply:
Makes sense.